src/single_net/GenGuide.cpp

```diff
@@ -135,32 +135,32 @@ void GuideGenerator::patchVioCells() {
             int x = dir == X ? gridline : cp;
             int y = dir == X ? cp : gridline;
 
-            double cellUsage = grDatabase.getCellUsage({layerIdx, x, y});
+            double cellRsrc = grDatabase.getCellResource({layerIdx, x, y});
 
-            if (cellUsage <= 0) {
+            if (cellRsrc <= 0) {
                 guideGenStat.vioCellNum++;
 
                 gr::GrBoxOnLayer patch(layerIdx, utils::IntervalT<int>(x), utils::IntervalT<int>(y));
 
                 for (int g = gridline - queryWidth; g <= gridline + queryWidth; g++) {
                     if (g < 0 || g >= grDatabase.getNumGrPoint(dir)) continue;
 
-                    double curCellUsage = dir == X ? grDatabase.getCellUsage({layerIdx, g, y})
-                                                   : grDatabase.getCellUsage({layerIdx, x, g});
-                    if (curCellUsage <= 0) continue;
-                    cellUsage += curCellUsage;
+                    double curcellRsrc = dir == X ? grDatabase.getCellResource({layerIdx, g, y})
+                                                   : grDatabase.getCellResource({layerIdx, x, g});
+                    if (curcellRsrc <= 0) continue;
+                    cellRsrc += curcellRsrc;
                     patch[dir].Update(g);
                 }
 
-                if (cellUsage > 0) {
+                if (cellRsrc > 0) {
                     const vector<int> layers = {layerIdx + 1, layerIdx - 1};
                     for (auto l : layers) {
                         if (l >= database.getLayerNum() || l <= 1) continue;
 
                         bool vioFree = true;
                         for (int x = patch.lx(); x <= patch.hx() && vioFree; x++)
                             for (int y = patch.ly(); y <= patch.hy() && vioFree; y++)
-                                if (grDatabase.getCellUsage({l, x, y}) <= 0) vioFree = false;
+                                if (grDatabase.getCellResource({l, x, y}) <= 0) vioFree = false;
```

Comment:
Always add curly brackets in `for` or `if` blocks
```suggestion
                                if (grDatabase.getCellResource({l, x, y}) <= 0) {
                                    vioFree = false;
                                }
```